### ISSUE TITLE
Cannot uninstall Bruno

### ISSUE THREAD

**Issue by zrpfpr** [usebruno/bruno#300]
Affected version: 0.17.0
OS: Windows 10 

Installed just to test its operation. When trying to uninstall a message is displayed indicating that the program is open, but there is no process running (maybe the own uninstaller??):

![image](https://github.com/usebruno/bruno/assets/6117119/f8436613-f623-4209-8756-142b0ce52dd1)

After click on Accept, a new error message shows:

![image](https://github.com/usebruno/bruno/assets/6117119/97ca878b-2493-49a9-af58-0d9b73bf2b7e)

**Comment by helloanoop** [comment]
Strange. We package using electron builder. This should work out of the box. 

I use Mac, so I can't test this on my system. Perhaps someone from the community with windows OS can test and confirm if they are facing the same issue when attempting to uninstall.

If nothing works, I suggest a restart and try uninstalling it.

**Comment by zrpfpr** [comment]
Oh, yes. I tried to restart, but I have the same error. 

Thanks!

**Comment by dcampagnone** [comment]
Tested uninstalling version 0.17.0 on Windows 11 and it worked fine.

**Comment by l0new4nder3r** [comment]
I could uninstall bruno version 0.16.6, on windows 10 without any issue.
_Windows settings, Applications and features, Bruno, Uninstall, OK_

@zrpfpr have you tried restarting your computer/windows session, then uninstalling again?

**Comment by zrpfpr** [comment]
Yes, I tried but I've got same error after several reboots... Only in safe mode finally I could uninstall the app (invoke the Un_A.exe located in AppData\tmp\~.nsu folder). 

After some research, apparently the error is related with electron builder and CHECK_APP_RUNNING flag in recent versions, but I could't find the proper solution. Anyway... The problem solves using the Un_A.exe file in safe mode and cmd session, if anybody has the same problem in the future. Greetings.

**Comment by jrnewton** [comment]
I'm getting this problem with 0.16.3 on Windows 10.  

![image](https://github.com/usebruno/bruno/assets/35567232/cab92c74-6ca2-455f-ad34-e29a5d760d68)

@zrpfpr - Un_A.exe doesn't exist anywhere under C:\Users\<Username>\.  Was it introduced in a later build?

**Comment by mmorton** [comment]
I was running into this same issue on version 0.22.0 when I tried to update. I got around it by going to `%LOCALAPPDATA%\Programs\bruno`, deleting `Bruno.exe` and then running `Uninstall Bruno.exe`.

**Comment by ForgottenIce** [comment]
I am having the same problem when trying to install (or uninstall) on Windows.
![image](https://github.com/usebruno/bruno/assets/38312721/89b58cc4-ec8b-4b58-a365-d106bf8ef888)
After a bit of research, I think the problem is related to [this issue](https://github.com/electron-userland/electron-builder/issues/6865#issue-1235865036) found on the electron-builder repo. Seems to be a problem with electron-builder v23.x. People are suggesting downgrading electron-builder to v22.x.

as @mmorton mentioned, deleting `Bruno.exe` and then running the installer (or uninstaller) fixes the issue, but it's quiet annoying having to do this step every time a new version arrives.

I tested this with chocolatey and it was the same result.

**Comment by iki** [comment]
@zrpfpr thanks, removing `%LOCALAPPDATA%\Programs\bruno\bruno.exe` works, it's an installer bug, system informer show the app is NOT running, so it's a false positive detection

**Comment by amandaguthrie** [comment]
Ran into this error while uninstalling 1.17 to upgrade to 1.18. Thanks to the other commenters! Removing `%LOCALAPPDATA%\Programs\bruno\bruno.exe` and then retrying worked for me as well.

**Comment by drizztguen77** [comment]
This is still a problem and the same solution of deleting the .exe file still works.

**Comment by anusree-bruno** [comment]
It’s been a few years, and Bruno has gone through many updates since then. This issue has become stale, so I’ll go ahead and close it.

If you’re still experiencing this issue, please create a new ticket with the relevant details and steps to reproduce it.
